netket/vqs/mc/kernels.py

```diff
@@ -68,7 +68,7 @@ def local_value_kernel_jax(
     """
     σp, mel = O.get_conn_padded(σ)
     logpsi_σ = logpsi(pars, σ)
-    logpsi_σp = logpsi(pars, σp)
+    logpsi_σp = logpsi(pars, σp.reshape(-1, σp.shape[-1])).reshape(σp.shape[:-1])
```

Comment:
https://chat.openai.com/share/eefd3861-16f7-4850-ac63-998784264c7b

Reply:
not really, there is a call to logpsi in between

Reply:
You should reshape also the \sigma, not only the \sigmap

Reply:
For the sake of readability can we define an auxiliary small function that does this reshaping? Something like 'at_worst1d' (similar to atleast1d)

Reply:
@jwnys the chains should already be flattened [here](https://github.com/netket/netket/blob/fe7629b996250b0661e78fb5f79e83648d1bc3c3/netket/vqs/mc/mc_state/expect.py#L133-134)